packages/base/any/kernels/5.10-lts/patches/0027-net-Add-Ethernet-2500Base-T-support-for-Marvell-10G.patch

```diff
@@ -0,0 +1,62 @@
+From b9e2a77fc45c7550764a7a122c6dd791b551a884 Mon Sep 17 00:00:00 2001
+From: Elad Nachman <enachman@marvell.com>
+Date: Wed, 28 Jul 2021 13:40:36 +0300
+Subject: [PATCH 4/4] net: Add Ethernet 2500Base-T support for Marvell 10G PHY
+
+This patch adds Ethernet 2500 Base-T support to the Marvell 10G PHY
+driver.
+Tested on Marvell A7K DB platform.
+
+Signed-off-by: Elad Nachman <enachman@marvell.com>
+---
+ drivers/net/phy/marvell10g.c | 11 ++++-------
+ 1 file changed, 4 insertions(+), 7 deletions(-)
+
+diff --git a/drivers/net/phy/marvell10g.c b/drivers/net/phy/marvell10g.c
+index 1901ba277413..96e96191cf2a 100644
+--- a/drivers/net/phy/marvell10g.c
++++ b/drivers/net/phy/marvell10g.c
+@@ -263,10 +263,6 @@ static int mv3310_power_up(struct phy_device *phydev)
+ 	ret = phy_clear_bits_mmd(phydev, MDIO_MMD_VEND2, MV_V2_PORT_CTRL,
+ 				 MV_V2_PORT_CTRL_PWRDOWN);
+ 
+-	if (phydev->drv->phy_id != MARVELL_PHY_ID_88X3310 ||
+-	    priv->firmware_ver < 0x00030000)
+-		return ret;
+-
```

Comment:
Why is this removed?

Reply:
This is an old limitation. Today there is a much greater variety of 10G copper PHY chips by Marvell today than just the 88X3310. Please see the updated list here: https://www.marvell.com/products/ethernet-phys.html .

Reply:
Please make it a separate commit, and update the commit message. When are you going to send it upstream?